relay-server/src/actors/events.rs

```diff
@@ -1165,6 +1187,8 @@ impl EventProcessor {
 
             self.finalize_event(&mut state)?;
 
+            self.sample_event(&mut state)?;
```

Comment:
Note: We could argue that if filters run after store processing, so should sampling. That said, we need to ensure that external Relays can both sample and filter, and for that reason it makes sense to keep it here and look at refactoring filters. 

cc @tonyo since you brought this up in the past.

Reply:
ok